framework/src/task_manager/task_manager.c

```diff
@@ -156,6 +158,30 @@ static int tm_handle_private_queue(char *q_name, tm_response_t *msg)
 	return OK;
 }
 
+static int tm_check_msg_filter(int msg, int handle)
+{
+	int ret = ERROR;
+	if ((msg & TASK_MSG_FILTER(handle)) == msg) {
```

Comment:
you'd better change like below.
(msg & TASK_MSG_FILTER(handle))
and I think ret value is not needed